internals/cli/cmd_enter.go

```diff
@@ -160,7 +160,7 @@ func (cmd *cmdEnter) Execute(args []string) error {
 	case runStop = <-runReadyCh:
 	case runPanic := <-runResultCh:
 		if runPanic == nil {
-			panic("internal error: pebble daemon stopped early")
+			panic("internal error: daemon stopped early")
```

Comment:
In case of multiple services wouldn't the program name be useful in a panic?

Reply:
That's fine -- I'd be happy to reinstate it as `{{.DisplayName}}`.